userguide/snippets/es60.rst

```diff
@@ -191,7 +191,11 @@ the top cover. Slide the top cover forward, then lift it off.
 Install Drive Trays
 ^^^^^^^^^^^^^^^^^^^
 
-Drive trays are used to mount drives in the array.
+Drive trays are used to mount drives in the array. These trays have
+two status lights for drives: green and amber. The drive tray light is
```

Comment:
This kind of switches back and forth between there being two lights and a single one.  Are there really two LEDs, or is it a single dual-color LED?